### PR TITLE
ZEN-25459 add checking permission to the trigger list

### PR DESCRIPTION
**Pull request by ssOlexBaiko**
Additional fix for filtering triggers from the trigger list.
See [Jira](https://jira.zenoss.com/browse/ZEN-25459)

### CODE REVIEW

File: Products/Zuul/facades/triggersfacade.py [hunk 1]
```diff
@@ -321,10 +321,13 @@ def getTriggerList(self):
         triggerList = []
         if 'triggers' in trigger_set:
             for t in trigger_set['triggers']:
-                triggerList.append(dict(
-                    uuid = t['uuid'],
-                    name = t['name']
-                ))
+                user = getSecurityManager().getUser()
```
Comment (glujan) [inline Products/Zuul/facades/triggersfacade.py]: Is this a cheap enough operation to do it in _inside_ `for` loop?

Comment (ssOlexBaiko) [inline Products/Zuul/facades/triggersfacade.py]: Ah, sure
I`ll move it outside

Comment (glujan) [inline Products/Zuul/facades/triggersfacade.py]: Sorry for double comment

File: Products/Zuul/facades/triggersfacade.py [hunk 2]
```diff
@@ -319,12 +319,15 @@ def getTriggerList(self):
         response, trigger_set = self.triggers_service.getTriggers()
         trigger_set = to_dict(trigger_set)
         triggerList = []
+        user = getSecurityManager().getUser()
         if 'triggers' in trigger_set:
             for t in trigger_set['triggers']:
```
Comment (glujan) [inline Products/Zuul/facades/triggersfacade.py]: Also this `if` before `for` is redundant, you can do: `for t in trigger_set.get('triggers', [])`

**Pull request merged**